### ISSUE TITLE
Pokesniper Was detected by PokeGo

### ISSUE THREAD

**Issue by judgejuctice** [pokesniperandroid/PokeSniper-Android#218]
![capture _2017-04-21-17-11-29](https://cloud.githubusercontent.com/assets/27870968/25297019/4e4644cc-26b9-11e7-9544-ca9997a9f7c8.png)
I got this as soon as i longed on 30mins after I sniped a porygon in philly.  the question is now should i change my Home Location to the spot im sniping just to be safe?

**Comment by jawknee209** [comment]
I got that to and I haven't even sniped like 2 weeks

**Comment by banjo97** [comment]
i got this as well, and i payed for my hashkey so now its useless???

**Comment by judgejuctice** [comment]
I might just change my home location and then wait 1 hour

**Comment by kkthedoc** [comment]
Niantic is coming to Sniper. Even after changing passwd as they suggest, you'll get this msg, even if you haven't done it or done it in a while. Btw, the site is down? Nothing appears on the PS page.

**Comment by judgejuctice** [comment]
I dont need the PS site i have a pokemap locater site that tells me the Longitude and Latitude i juat hope they dont go down

**Comment by jawknee209** [comment]
When what do you use to snipe  them after you get the lat and long

**Comment by judgejuctice** [comment]
Yes i Philadelphia and New York City have a map where you can see Commons to Very Rares all i do is click on the Pokemon and it will come up with a Map button and then take me to Google Maps Then Google Map will 80% of the give me Longitude and Latitude

**Comment by sapukasibersih** [comment]
Until now..still got the warning from niantic..is it save if still use?

**Comment by pokesniperandroid** [comment]
If you want to be on safe side, use with secondary account or test account.
